courses/free-courses-en.md

```diff
@@ -1452,6 +1452,7 @@
 * [Stanford Cryptography II](https://www.coursera.org/course/crypto2) - Dan Boneh
 * [Start Ethical Hacking with Parrot Security OS (Alt. to Kali)](https://www.udemy.com/course/ethical-hacking-with-parrot-security-os) - Seyed Farshid Miri (Udemy)
 * [The Complete Cyber Security & Hacking Course](https://academy.ehacking.net/p/cyber-security-training-hacking-course) - INSEC-TECHS (EH Academy)
+* [TJ_Null’s OSCP Prep](https://youtube.com/playlist?list=PLidcsTyj9JXK-fnabFLVEvHinQ14Jy5tf&feature=shared) - TJ_Null(HackTheBox)
```

Comment:
```suggestion
* [TJ_Null’s OSCP Prep](https://youtube.com/playlist?list=PLidcsTyj9JXK-fnabFLVEvHinQ14Jy5tf&feature=shared) - TJ_Null
```
I don't think this applies as the course is on YouTube, albeit showing Hack the Box. If it does end up being something we want, there should be a space before the opening parenthesis.

Reply:
I think it might be important to add the tag because the labs performed in the YouTube videos are created by Hack The Box.